Source/DafnyDriver/Commands/CommandRegistry.cs

```diff
@@ -45,6 +45,7 @@ static class CommandRegistry {
     AddCommand(new RunCommand());
     AddCommand(new BuildCommand());
     AddCommand(new TranslateCommand());
+    AddCommand(new StabilityCommand());
```

Comment:
I think I'd prefer this to be a set of options to the `verify` command rather than a separate command, though I'm open to arguments to the contrary. At the very least, though, it will be valuable to be able to analyze stability (or whatever we call it) in the context of any other set of verification options, so this command (if it's a separate command) should accept all (or almost all) of the same options as the `verify` command.

Reply:
> this command should accept all (or almost all) of the same options as the verify command.

Agreed 

> I think I'd prefer this to be a set of options to the verify command rather than a separate command, though I'm open to arguments to the contrary

Why do you prefer that?

Having them be separate commands means that the options that only relate to stability, of which I think there can be quite a few, can be shown at the top of the help for the stability command, which is makes it easier to use.

The stability command also has different output than the verify command since it also includes a statistics report, which I would print to stdout before any diagnostics, and when printing diagnostics is might be good to include the random seed required to get that diagnostic.